tutorials/introductory/animation_tutorial.py

```diff
@@ -0,0 +1,225 @@
+#   artists in variables so that they can be updated later on during the
+#   animation.
+# - Create an animation function that updates the data in each artist to
+#   generate the new frame at each function call.
+# - Create a `.FuncAnimation` object with the `.Figure` and the animation
+#   function, along with the keyword arguments.
+# - Use `.animation.Animation.save` or `.pyplot.show` to save or show the
+#   animation.
+#
+# The update function uses the ``set_*`` function for different artists to
+# modify the data. The following table shows a few plotting methods, the artist
+# types they return and some methods that can be used to update them.
+#
+# =================  =============================  ===========================
+# Plotting method    Artist                         Set method
+# =================  =============================  ===========================
+# `.Axes.plot`       `.lines.Line2D`                `.lines.Line2D.set_data`
+# `.Axes.scatter`    `.collections.PathCollection`  `.collections.PathCollecti\
+#                                                   on.set_offsets`
+# `.Axes.imshow`     `.image.AxesImage`             ``AxesImage.set_data``
+# `.Axes.annotate`   `.text.Annotation`             `.text.Annotation.update_p\
+#                                                   ositions`
+# `.Axes.barh`       `.patches.Rectangle`           `.Rectangle.set_angle`,
+#                                                   `.Rectangle.set_bounds`,
+#                                                   `.Rectangle.set_height`,
+#                                                   `.Rectangle.set_width`,
+#                                                   `.Rectangle.set_x`,
+#                                                   `.Rectangle.set_y`
+#                                                   `.Rectangle.set_xy`
+# `.Axes.fill`       `.patches.Polygon`             `.Polygon.set_xy`
+# `.patches.Circle`  `.patches.Ellipse`             `.Ellipse.set_angle`,
+#                                                   `.Ellipse.set_center`,
+#                                                   `.Ellipse.set_height`,
+#                                                   `.Ellipse.set_width`
+# =================  =============================  ===========================
+#
+# Covering the set methods for all types of artists is beyond the scope of this
+# tutorial but can be found in their respective documentations. An example of
+# such update methods in use for `.Axes.scatter` is as follows.
+
+
+fig, ax = plt.subplots()
+t = np.linspace(-4, 4, 400)
+a, b = 3, 2
+delta = np.pi / 2
+
+scat = ax.scatter(np.sin(a * t[0] + delta), np.sin(b * t[0]), c="b", s=2)
+ax.set_xlim(-1.5, 1.5)
+ax.set_ylim(-1.5, 1.5)
+
+
+def update(frame):
+    # .set_offsets also resets the offset data for the entire collection with
+    # the new values. Therefore, to also carry forward the previously
```

Comment:
I don't think I'd call it 'resetting', as there is no way to set just a single value.
```suggestion
    # .set_offsets replaces the offset data for the entire collection with
    # the new values. Therefore, to also carry forward the previously
```